### PR TITLE
Fixes and simplifies the 'network' test

### PR DESCRIPTION
**Pull request by inetic**
Was only panicking on CI machines, and even then the test
resulted with OK for some reason.

[Review on Reviewable](https://reviewable.io/reviews/maidsafe/crust/339)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @vinipsmaker

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: src/service.rs [hunk 1]
```diff
@@ -585,155 +542,96 @@ mod test {
 
     #[test]
     fn network() {
+        //extern crate env_logger;
```
Comment (vinipsmaker) [inline src/service.rs]: remove the commented code?

**Comment by vinipsmaker** [comment]
All else seems fine.

File: src/service.rs (hunk 1 above)
Comment (inetic) [inline src/service.rs]: Done

**Pull request merged**